articles/native-platforms/cordova.md

```diff
@@ -119,3 +119,14 @@ cp plugins/cordova-plugin-inappbrowser/src/android/* platforms/android/[yourAppN
 #### Get a blank page with an OK after signin
 
 This means that the `InAppBrowser` plugin wasn't installed successfully by Cordova. See the previous section to learn how to solve this.
```

Comment:
Should the "See the previous section to learn how to solve this." still be here since you have shown below how to solve?